### DISCUSSION TITLE
Website does not allow files to be posted with underscores

### DISCUSSION THREAD

**Discussion by ncromes** [CMSgov/hospital-price-transparency#123] [Q&A] [answered]
Our content management system does not allow the use under scores_ .  Is it ok to use a dash- instead of underscores for the MRF file name?

**Comment by CarlsonErin** [comment, marked as answer]
Hospitals must use the following CMS naming convention as specified in the regulations at 45CFR § 180.50(d)(5) for the machine-readable file:
<ein>_<hospital- name>_standardcharges.[json| |csv] in which the EIN isthe Employer Identification Number of the hospital, followed by the hospital name, followed by “standardcharges” followed by the hospital’s chosen file format (84 FR 65562).
